src/Catalyst.Core.Lib.Tests/UnitTests/Cryptography/Rfc8410Test.cs

```diff
@@ -0,0 +1,113 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Linq;
+using System.Security;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Options;
+using Catalyst.Core.Lib.Config;
+using Catalyst.Core.Modules.Keystore;
+using Catalyst.TestUtils;
+using Makaretu.Dns;
+using MultiFormats;
+using NSubstitute;
+using Org.BouncyCastle.Crypto.Parameters;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Catalyst.Core.Lib.Tests.UnitTests.Cryptography
+{
+    public class Rfc8410Test : FileSystemBasedTest
+    {
+        private readonly KeyStoreService _keyStoreService;
+        
+        public Rfc8410Test(ITestOutputHelper output) : base(output)
+        {
+            var dfsOptions = new DfsOptions(new BlockOptions(), new DiscoveryOptions(), new RepositoryOptions(FileSystem, Constants.DfsDataSubDir), Substitute.For<KeyChainOptions>(), Substitute.For<SwarmOptions>(), Substitute.For<DotClient>());
+            _keyStoreService = new KeyStoreService(dfsOptions)
+            {
+                Options = dfsOptions.KeyChain
+            };
+            var securePassword = new SecureString();
+
+            "mypassword".ToList().ForEach(c => securePassword.AppendChar(c));
+
+            securePassword.MakeReadOnly();
+            _keyStoreService.SetPassphraseAsync(securePassword).ConfigureAwait(false);
+        }
+        
+        [Fact]
+        public async Task ReadPrivateKey()
+        {
+            const string alice1 = @"-----BEGIN PRIVATE KEY-----
+MC4CAQAwBQYDK2VwBCIEINTuctv5E1hK1bbY8fdp+K06/nwoy/HU++CXqI9EdVhC
+-----END PRIVATE KEY-----
+";
+            var _ = await _keyStoreService.ImportAsync("alice1", alice1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)